evap/grades/migrations/0009_translate_description.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.1 on 2016-01-04 18:32
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('grades', '0008_translate_description'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='gradedocument',
+            unique_together=set([('course', 'description_de'), ('course', 'description_en')]),
+        ),
+        migrations.RemoveField(
+            model_name='gradedocument',
+            name='description',
+        ),
+    ]
```

Comment:
i'll think about the migration tomorrow. just a quick thought, @janno42 is it possible to just rename description to description_de, create a new description_en and add a unique together on that one? (maybe make description a CharField beforehand)